core/drand_daemon_control.go

```diff
@@ -145,6 +145,22 @@ func (dd *DrandDaemon) Shutdown(ctx context.Context, in *drand.ShutdownRequest)
 	return &drand.ShutdownResponse{Metadata: metadata}, nil
 }
 
+// ResurrectBeacon
+func (dd *DrandDaemon) ResurrectBeacon(ctx context.Context, in *drand.ResurrectBeaconRequest) (*drand.ResurrectBeaconResponse, error) {
```

Comment:
can we call this 'ReloadBeacon' so that it's consistent in naming between the method in in drand that's called, and the CLI command we use to refer to that action?

Reply:
Done!